internal/cloudsql/refresh.go

```diff
@@ -180,6 +181,17 @@ func genVerifyPeerCertificateFunc(cn connName, pool *x509.CertPool) func(rawCert
 	}
 }
 
+// newRefresher creates a Refresher.
+func newRefresher(timeout time.Duration, interval time.Duration, burst int, svc *sqladmin.Service) refresher {
+	return refresher{
+		timeout:       timeout,
+		clientLimiter: rate.NewLimiter(rate.Every(interval), burst),
+		client:        svc,
+	}
+}
```

Comment:
Why are we using an initializer here?

Reply:
Two reasons: I'm don't want to type all these fields out in test, and it seems better to hide the rate limiter inside the refresher.

Reply:
I'll downgrade this to a nit, but I think it's usage from `instance.go` is harder to interpret with the constructor.

Reply:
I agree the initializer isn't super easy to read. Let's let this sit for awhile and revisit it when the refresher changes. It's possible it's better to just manually configure the refresher in test rather than introduce an initializer.